page-Objects/create-blog-post.pageObject.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+class CreateBlogPost {
+
+	constructor() {
+		this.name = element(by.id('element-type-blog-name'));
+		this.title = element(by.id('element-type-blog-title'));
+		this.body = element(by.className('note-editable'));
+		this.save = element(by.id('element-type-blog-submit'));
+	}
+
+};
+
```

Comment:
I recommend that the directory name is:
* page-objects
or
* pageObjects
but not
page-Objects